gammapy/data/event_list.py

```diff
@@ -142,7 +142,7 @@ def read(cls, filename, hdu="EVENTS", checksum=False, **kwargs):
         """
         from gammapy.data.io import EventListReader
 
-        return EventListReader(hdu, checksum).read(filename)
+        return EventListReader(hdu, checksum).read(filename, **kwargs)
```

Comment:
Thanks @LeanderSchlegel . Here I would simply have removed the `**kwargs` argument altogether.

And possibly add an explicit argument `format` for future usage.